libobs/obs-display.c

```diff
@@ -273,11 +274,11 @@ void render_display(struct obs_display *display)
 
 		pthread_mutex_unlock(&display->draw_callbacks_mutex);
 
-		render_display_end();
-
 		GS_DEBUG_MARKER_END();
 
 		gs_present();
+
+		render_display_end();
```

Comment:
It doesn't make sense to present a display before it's rendered.

Reply:
render end This represents the original state